### DISCUSSION TITLE
Who is Contributing Apache Kyuubi?

### DISCUSSION THREAD

**Discussion by yaooqinn** [apache/kyuubi#3383] [Show and tell]
![](https://user-images.githubusercontent.com/8326978/187616678-1ec62888-5ea8-4fc5-ad9b-06693f9020e9.svg)

## Correct me if you find you are missing!

**Comment by minyk** [comment]
@yaooqinn thanks, my boss (and his boss and his boss, et al) really likes this picture.

**Reply by yaooqinn** [reply]
Hi @minyk, if you, your boss, and your boss's boss like, you can comment on #925 , and we can add your logo to our [main page](https://kyuubi.apache.org/) too

**Reply by minyk** [reply]
@yaooqinn I'll write the information soon, thank you.

**Reply by yaooqinn** [reply]
coooool
